src/deepsparse/benchmark_model/README.md

```diff
@@ -14,103 +14,123 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Benchmarking
+# Benchmarking ONNX Models 📜
 
-The command-line (CLI) tool `deepsparse.benchmark` is meant to simplify benchmarking and be the central code path for benchmarking the engine on ONNX or SparseZoo models.
+`deepsparse.benchmark` is a command-line (CLI) tool for benchmarking the DeepSparse Engine with ONNX models. The tool will parse the arguments, download/compile the network into the engine, generate input tensors, and execute the model depending on the chosen scenario. By default, it will choose a multi-stream or asynchronous mode to optimize for throughput.
 
 ## Quickstart
 
-After you have installed `deepsparse` in your Python environment, the benchmark tool is available on your CLI. It is simple to run expressive model benchmarks on the DeepSparse Engine with minimal parameters.
-
-To run a sparse FP32 MobileNetV1 at batch size 16 for 10 seconds for throughput using 8 streams of requests:
+After `pip install deepsparse`, the benchmark tool is available on your CLI. For example, to benchmark a dense BERT ONNX model fine-tuned on the SST-2 dataset where the model path is the minimum input required to get started, run:
 
 ```
-deepsparse.benchmark zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate --batch_size 8 --time 10 --scenario async --num_streams 8
+deepsparse.benchmark zoo:nlp/text_classification/bert-base/pytorch/huggingface/sst2/base-none
 ```
+__ __
+## Usage
 
-To run a sparse quantized INT8 6-layer BERT at batch size 1 for latency:
+In most cases, good performance will be found in the default options so it can be as simple as running the command with a SparseZoo model stub or your local ONNX model. However, if you prefer to customize benchmarking for your personal use-case, you can run `deepsparse.benchmark -h` or `--help` to view your usage options:
```

Comment:
```suggestion
In most cases, good performance will be found in the default options so it can be as simple as running the command with a SparseZoo model stub or your local ONNX model. However, if you prefer to customize benchmarking for your personal use case, you can run `deepsparse.benchmark -h` or with `--help` to view your usage options:
```